tests/common/test_transaction.py

```diff
@@ -1023,7 +1018,6 @@ def test_create_transfer_transaction_multiple_io(user_pub, user_priv,
                     'owners_before': [
                         user2_pub
                     ],
-                    'fid': 1,
                     'fulfillment': None,
                     'input': {
                         'txid': tx.id,
```

Comment:
This really depends on what we want to test for.

If rethink and mongo provide guarantees (e.g. states it somewhere) for the order of arrays to be kept, then I don't think we need to consider anything more than just model-level tests.

Would just taking a transaction dict with multiple conditions / fulfillments, modifying this dict to have different orders in each, re-serializing each of the dicts to `Transaction` instances, and then checking that their hashes and condition / fulfillment orders are still different be enough?